scripts/zowe-install-existing-plugin.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Requires passed in:
+# ZOWE_ROOT_DIR - install root directory
+# PLUGIN_ID - id of the plugin - e.g. "org.zowe.zlux.auth.apiml"
+# PLUGIN_DIR - absolute path to the directory with the plugin
+
+ZOWE_ROOT_DIR=$1
+PLUGIN_ID=$2
+PLUGIN_DIR=$3
+
+if [ "$#" -ne 3 ]; then
+  echo "Usage: $0 ZOWE_INSTALL_ROOT_DIRECTORY PLUGIN_ID PLUGIN_DIR\ne.g. install-existing-plugin.sh \"~/zowe\" \"org.zowe.plugin.example\" \"~/zowe/myplugins/example\"" >&2
+  exit 1
+fi
+if ! [ -d "$1" ]; then
+  echo "$1 not a directory, please provide the full path to the root installation directory of Zowe" >&2
+  exit 1
+fi
+if ! [ -d "$3" ]; then
+  echo "$3 not a directory, please provide the full path to the directory with the plugin" >&2
+  exit 1
+fi
+
+zluxserverdirectory='zlux-example-server'
```

Comment:
This has changed to 'zlux-app-server' in https://github.com/zowe/zowe-install-packaging/pull/241